README.md

```diff
@@ -120,41 +124,193 @@ Each flavor represents a pair of programming language and builds tool. All the u
 
 ### Core flavor
 
-Create-cycle-app come packed with 1 core flavor:
+Create-cycle-app comes pre-packaged with this core flavor:
 
 * [cycle-scripts](https://github.com/cyclejs-community/create-cycle-app/blob/master/packages/cycle-scripts)
 
-### Custom flavour
+### Using a custom flavor
 
 Custom flavors allow generating starting projects to fulfil specific needs.
-They can be published to npm, or being used locally via the create-cycle-app CLI.
+They can be published to npm or a git repo or even be referenced locally (esp. useful when you create your _own_ custom flavor - see below)
 
 #### How to use custom flavors
 
 ##### Providing your own
-When creating a project, you can inform which flavor you want to use with the `--flavor` flag:
+When creating a project, you can specify which flavor to use with the `--flavor` flag:
 
 ```sh
 $ create-cycle-app <name> --flavor <flavor>
 ```
-Some examples of how a flavor could be specified:
 
+The flavor will be installed via `npm install` and can therefore reference any flavor resource installable via `npm`
+
+Some examples of how a flavor can be specified:
+
+*npm name*
+
+```sh
+$ create-cycle-app my-app --flavor cycle-scripts-one-fits-all
 ```
-$ create-cycle-app my-app --flavor cycle-scripts-es-webpack
 
-$ create-cycle-app my-app --flavor cycle-scripts-es-webpack@x.y.z
+*npm name with specific version*
 
-$ create-cycle-app my-app --flavor ./relative/path/to/cycle-scripts-es-webpack
+```sh
+$ create-cycle-app my-app --flavor cycle-scripts-one-fits-all@1.1.0
+```
+
+*github account/repo*
+
+```sh
+$ create-cycle-app my-app --flavor github:my-account/my-cycle-flavor
+```
+
+*reference to local file path*
+
+```sh
+$ create-cycle-app my-app --flavor ./relative/path/to/my-cycle-flavor
 ```
 
 ##### Flavors:
 
 - Check [available flavors](https://github.com/cyclejs-community/create-cycle-app-flavors)
 
+### Creating a custom flavor
```

Comment:
This part is definitely interesting but I found it a bit unclear atm. Why not focusing on explaining better  how to create a flavor and less on lerna technicalities? (We use lerna for managing this mono-repo and the flavors mono-repo, but i'm not too sure is of any relevance for a flavor creator) ..apart the sparse checkout mention, good one!